### ISSUE TITLE
E126 not detected by pycodestyle but by flake8

### ISSUE THREAD

**Issue by spaceone** [PyCQA/pycodestyle#1097]
foo.py
```
from foo import (  # noqa: E402
        bar, baz, blub
)
```
^ 8 spaces

```
$ pycodestyle --select E126 foo.py
$ flake8 --select E126 foo.py
foo.py:2:9: E126 continuation line over-indented for hanging indent
```

This results in `autopep8` to not fix that error while flake8 complains about it.

**Comment by asottile** [comment]
`pycodestyle` has no concept of per-code `noqa` -- this is a flake8 feature.  as such `pycodestyle` is ignoring the whole line via the `# noqa` -- you'll see for example:

```console
$ flake8 --select=E126 t.py
t.py:2:9: E126 continuation line over-indented for hanging indent
$ pycodestyle --select=E126 t.py
t.py:2:9: E126 continuation line over-indented for hanging indent
$ cat t.py 
from foo import (  # wat: E402
        bar, baz, blub
)
```

**Comment by spaceone** [comment]
but `pycodestyle` detects any `noqa` statement and ignore the whole context. If you close this as wontfix, can you provde a command line option `--disable-noqa` then? So that `autopep8` is able to use `--disable-noqa` then and fix it nevertheless?

**Comment by asottile** [comment]
that wouldn't work either -- then you'd have the opposite problem "why is autopep8 fixing something that I've marked noqa"

**Comment by spaceone** [comment]
no, the user could decide that on their own by giving `autopep8` the info that it should use that disable-noqa feature and maybe also just `--select` certain errors.

I am also not a user of pycodestyle due to the bad `noqa` implementation, which is handled better by `flake8` - but with using `autopep8` one doesn't have the choice.

**Comment by asottile** [comment]
maybe I wasn't clear -- how about a concrete example -- what would you expect autopep8 to do for:

```python
from foo import (
        bar,  # noqa: E126
)
```

**Comment by spaceone** [comment]
to convert it - nevertheless the noqa-staement - into:
```
from foo import (
    bar,  # noqa: E126
)
```

**Comment by asottile** [comment]
yeah I suspect you'd have a hard time selling that to autopep8 -- it would be a breaking change there and it's documented and tested to work opposite to that: https://github.com/hhatto/autopep8/blob/e7b494456c5ffd0624d56d9c996c17e6529f18a1/test/suite/noqa.py

**Comment by spaceone** [comment]
that's why I suggest to make it configurable both in pycodestyle and in autopep8.
